### PR TITLE
simplify code

### PR DESCRIPTION
**Pull request by playasim**
Simplified some code for numerical comparison.

### CODE REVIEW

**Comment by playasim** [comment]
@googlebot I signed it!

**Comment by playasim** [comment]
@googlebot I signed it!

**Comment by playasim** [comment]
@googlebot I fixed it.

**Comment by netdpb** [comment]
I'm not sure these changes are simpler to understand than what's there. `Math.min` and `Math.max` are better in my opinion when neither argument is a constant. Every time I look at `Math.min` or `Math.max` calls for pinning a range to a constant, I have to think a bit about what it means. And two of these changes mean that we're using simple comparison for one end of a range and min or max for the other.

Can you help me to understand why, in your opinion, these changes are helpful to readers?

Thanks!

**Comment by playasim** [comment]
> I'm not sure these changes are simpler to understand than what's there. `Math.min` and `Math.max` are better in my opinion when neither argument is a constant. Every time I look at `Math.min` or `Math.max` calls for pinning a range to a constant, I have to think a bit about what it means. And two of these changes mean that we're using simple comparison for one end of a range and min or max for the other.
> 
> Can you help me to understand why, in your opinion, these changes are helpful to readers?
> 
> Thanks!

As you said, when two arguments are not constant, `Math.max` or `Math.min` is easy to understand, so the modifications to the `toSafeNanos` method make sense; for the other two methods, I think using `Math.max` method is also very readable, because it is easy to see that our intention is to get the number that is greater than 0 or -1.

File: guava/src/com/google/common/math/PairedStatsAccumulator.java [hunk 1]
```diff
@@ -239,9 +239,6 @@ private static double ensureInUnitRange(double value) {
     if (value >= 1.0) {
       return 1.0;
     }
-    if (value <= -1.0) {
-      return -1.0;
-    }
-    return value;
+    else return Math.max(value, -1.0);
```
Comment (netdpb) [inline guava/src/com/google/common/math/PairedStatsAccumulator.java]: Could this whole method use `Doubles.constrainToRange()`?

That would be a more simplifying change.

File: guava/src/com/google/common/util/concurrent/Monitor.java [hunk 2]
```diff
@@ -1032,7 +1032,7 @@ private static long toSafeNanos(long time, TimeUnit unit) {
     long timeoutNanos = unit.toNanos(time);
     return (timeoutNanos <= 0L)
         ? 0L
-        : (timeoutNanos > (Long.MAX_VALUE / 4) * 3) ? (Long.MAX_VALUE / 4) * 3 : timeoutNanos;
+        : Math.min(timeoutNanos, (Long.MAX_VALUE / 4) * 3);
```
Comment (netdpb) [inline guava/src/com/google/common/util/concurrent/Monitor.java]: Similarly, what about using `Longs.constrainToRange()` here?

File: guava/src/com/google/common/math/PairedStatsAccumulator.java (hunk 1 above)
Comment (playasim) [reply guava/src/com/google/common/math/PairedStatsAccumulator.java]: Really clean and strong change, i will replace both with `constrainToRange()`

**Comment by netdpb** [comment]
OK, this looks fine. I'll copy the change internally, and it'll get mirrored out shortly.

**Pull request closed without merging**